packages/aws-cdk/lib/util/tables.ts

```diff
@@ -1,13 +1,43 @@
-import Table = require('cli-table');
+import colors = require('colors/safe');
+import table = require('table');
 
 export interface RenderTableOptions {
   colWidths?: number[];
 }
 
 export function renderTable(cells: string[][], options: RenderTableOptions = {}): string {
-  const head = cells.splice(0, 1)[0];
+  const columns: {[col: number]: table.ColumnConfig} = {};
 
-  const table = new Table({ head, style: { head: [] }, colWidths: options.colWidths });
-  table.push(...cells);
-  return table.toString();
+  if (options.colWidths) {
+    options.colWidths.forEach((width, i) => {
+      columns[i] = { width, useWordWrap: true } as any; // @types doesn't have this type
```

Comment:
Why do you need to cast to `any` here? You have the type, `table.ColumnConfig`, is this not the correct shape?